src/DrawerScreenNavigators.js

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {createStackNavigator} from '@react-navigation/stack';
+import LeessonsList from './screens/LessonsList';
+import Lesson from './screens/lessonBase';
+
+const Stack = createStackNavigator();
+
+const Level1Navigation = () => {
+  return (
+    <Stack.Navigator>
+      <Stack.Screen name='Overview' component={LeessonsList} />
+      <Stack.Screen name='Lesson 0' component={Lesson} options={{headerShown: true}} />
```

Comment:
call the name Level1Lesson because this screen will be used for ALL lessons.

Reply:
I'm not sure what you mean here @mark-yacoub